.changeset/tame-papayas-remain.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sap-ux/adp-tooling": patch
+"@sap-ux-private/preview-middleware-client": patch
```

Comment:
`@sap-ux/preview-middleware` is missing here.

Reply:
done